L1Trigger/TrackFindingTMTT/src/KFbase.cc

```diff
@@ -758,7 +758,7 @@ namespace tmtt {
           }
           break;
           //case 5:  // B1 B2 B3+B4 D1 D2 D3 D4/D5
-        case 5:  // B1 B2 B3 D1+B4 D2 D3 D4/D5
+        case 5:
```

Comment:
Why has the comment explaining which layers "case 5" corresponds to been commented out?

Reply:
I don't know why there are two comments on this case 5. I deleted one line.

Reply:
I suspect the commented out one was for the KF before maybe layers were introduced, so was kept in case we ever chose to disable them. The uncommented one should correspond to the current cfg, so should not be deleted.

Reply:
Comment is back